srcpkgs/dotool/files/80-dotool.rules

```diff
@@ -0,0 +1,2 @@
+# This allows users in group input to use dotool without root permissions.
+KERNEL=="uinput", GROUP="input", MODE="0660"
```

Comment:
this is something general enough that you should maybe include it upstream for other distro packages

Reply:
okay thank you I'll sleep on it.

Reply:
Do you think a more specific group would be better? I was thinking "**u**input" but maybe even "dotool"?

Reply:
input is a fairly standard name across distros. I think systemd distros have it by default, with the same purpose as void's input group

Reply:
in fact, there might already be a udev rule that does this maybe?

Reply:
If I remove the rule and reboot it doesn't work so I don't think there's a rule already.

My reasoning for a uinput group was just because access to /dev/uinput is less of a big deal than to /dev/input, and so you wouldn't need to give a user as big a permission.
Maybe I should do both? then it'd just work for everyone in input too.

Reply:
i'd go with the [existing system group](https://docs.voidlinux.org/config/users-and-groups.html#default-groups), input

Reply:
i followed your advice